lib/imapSimple.js

```diff
@@ -225,9 +261,49 @@ function connect(options, callback) {
     });
 }
 
+/**
+ * Returns a list of the parts.
+ * A part is either a message or an attachment.
+ * Parts are usually nested and numbered things like 1, 1.1, 1.2, ...
+ * Code taken from http://stackoverflow.com/questions/25247207/how-to-read-and-save-attachments-using-node-imap
+ */
+function findParts(struct, parts) {
```

Comment:
I'd argue this this one is slightly more useful than `decorateMessage`. It's useful in most cases where you want to fetch attachments. But happy to remove it too